compiler-rt/test/radsan/TestCases/test_continue_mode.cpp

```diff
@@ -0,0 +1,31 @@
+// RUN: %clangxx -fsanitize=realtime %s -o %t
+// RUN: env RADSAN_ERROR_MODE=continue %run %t 2>&1 | FileCheck %s
+// UNSUPPORTED: ios
+
+// FIXME: We should have the second "RUN" command be prefaced with "not", 
+// aka "not env RADSAN_ERROR_MODE=continue %run %t 2>&1 | FileCheck %s"
```

Comment:
Should we try to get the executable to exit with a nonzero return code if RADSan has detected an error in continue mode? One idea - I guess it might be possible to put some sort of class instance in the RADSan library that:

1. has lifetime beyond `main`,
2. maintains an atomic bool for whether RADSan ever detected a violation or not, and
3. calls `exit(EXIT_FAILURE)` in its destructor.

Alternative idea: we could put some checking function into the `.fini_array`, to be called after `main`, just like we insert some interceptor setup logic into the `.preinit_array` (which is called before `main`). This checking function could `exit(EXIT_FAILURE)` if RADSan ever errored in continue mode!

Reply:
Yep! You got the general sketch totally right. There is a lot of good examples of how the other sanitizers do this, and you'll see the base of it put down in this upcoming review for the flags.

Reply:
Cool, so should I wait for the next PR and resolve this comment thread for now?

Reply:
yep. It won't even be entirely fixed in that review, as to do "one thing per review" but you'll at least see the direction it's going.

The fix for it is a little involved, so I want to make sure these PRs stay focused on one task at a time. I'm hot on the trail though, and won't let this comment/problem linger for long.